libs/FirmwareKit/tests/FirmwareKit_test.cpp

```diff
@@ -38,11 +39,14 @@ class FirmwareKitTest : public ::testing::Test
 	std::string bin_update_path				  = "/tmp/update-v2.0.0.bin";
 	std::array<uint8_t, 6> bin_update_content = {0x61, 0x62, 0x63, 0x64, 0x65, 0x66};	// "abcdef"
 
-	FirmwareVersion default_version = FirmwareVersion {1, 0, 0};
-	FirmwareVersion current_version = FirmwareVersion {1, 2, 3};
-	FirmwareVersion update_version	= FirmwareVersion {2, 0, 0};
+	// ? Default made sense when we were reading from a file in case the file was not readable
+	// ? With the #define OS_VERSION, this is not needed anymore
+	// TODO (@YannLocatelli): Remove the file related tests?
+	Version default_version = Version {1, 2, 0};
+	Version current_version = Version {1, 2, 0};
+	Version update_version	= Version {2, 0, 0};
```

Comment:
`default_version` is used only when SD is not accessible. It is the lowest possible. As we stop to relate to SD, it is not anymore necessary.

Aside of this, `current_version` is an interesting one: it will need to be bumped too at each bump in that configuration. Do we need that?
Does the one that will be used in CI not sufficient?

Reply:
> default_version is used only when SD is not accessible. It is the lowest possible. As we stop to relate to SD, it is not anymore necessary.

Yes we can remove it.